### PR TITLE
BIC-5084: added Google Pay letter P to entry_method enum

### PR DESCRIPTION
**Pull request by jwebster-worldline**

### CODE REVIEW

**Review by wenfeiliang — APPROVED** [review]

File: data/na/swagger/1-0-5.yaml [hunk 1]
```diff
@@ -2076,7 +2076,7 @@ definitions:
         type: string
         maxLength: 1
         description: 'How the transaction was initiated. N = Standard Entry, A = Apple Pay, G = Android Pay, P = Google Pay, S = EMV Swipe, V = Visa Checkout, Y = Unimag Card Swiper, T = EMV Chip Tap, E = EMV Chip Dip'
```
Comment (TDScrumteam) [inline data/na/swagger/1-0-5.yaml:2078]: Just curious did we add google pay description in one of our previous tickets ?

Comment (jwebster-worldline) [reply data/na/swagger/1-0-5.yaml:2078]: I added Google Pay to the description when I was working on this ticket previously, a couple of weeks ago

Comment (TDScrumteam) [reply data/na/swagger/1-0-5.yaml:2078]: Ok thanks.

**Review by TDScrumteam — APPROVED** [review]
ok.

**Pull request merged**